Source/Path Configuration/PathConfigurationLoader.swift

```diff
@@ -20,7 +19,7 @@ final class PathConfigurationLoader {
         for source in sources {
             switch source {
             case .data(let data):
-                loadData(data)
+                loadData(data, for: .PathDataTemporaryURL)
```

Comment:
I think we could skip passing the url when not needed. See https://github.com/hotwired/turbo-ios/pull/216

Reply:
Merged your draft PR. Thanks!